### DISCUSSION TITLE
custom command Keybinding

### DISCUSSION THREAD

**Discussion by haithium** [kdheepak/taskwarrior-tui#553] [Q&A]
Is there a way to set keybindings to exclude custom commands? For example, can I set a keybinding to assign **priority** as **high** without using `m` + `priority:H`?

**Comment by kdheepak** [comment]
You can make a custom shortcut to set any task to any priority and map one of the keys to that shortcut.

**Reply by haithium** [reply]
Could you explain how?

**Reply by niderhoff** [reply]
https://kdheepak.com/taskwarrior-tui/configuration/advanced/#configure-user-defined-shortcuts
